### PR TITLE
Fixing backup server-var.yml

### PR DESCRIPTION
**Pull request by hkim823**

### CODE REVIEW

**Comment by hkim823** [comment]
Squashed commits

**Comment by jarv** [comment]
needs rebase

File: playbooks/roles/edx_ansible/tasks/deploy.yml [hunk 1]
```diff
@@ -34,11 +34,16 @@
     dest={{ COMMON_CFG_DIR }}/playbooks
     state=link
 
+- name: check to see if server vars already exists
+  stat: path={{ edx_ansible_var_file }}
+  register: edx_ansible_var_file_exists
+
 - name: create backup of server vars
   copy: >
     src={{ edx_ansible_var_file }}
```
Comment (feanil) [inline playbooks/roles/edx_ansible/tasks/deploy.yml:43]: This line should be deleted.

File: playbooks/roles/edx_ansible/tasks/deploy.yml [hunk 2]
```diff
@@ -34,11 +34,16 @@
...
     dest={{ edx_ansible_var_file }}.old
```
Comment (feanil) [inline playbooks/roles/edx_ansible/tasks/deploy.yml:44]: Let's use the 'path' alias instead of the 'dest' alias so that this is more clear.

**Pull request closed without merging**